janusgraph-test/src/test/java/org/janusgraph/graphdb/GraphDatabaseConfigurationInstanceIdTest.java

```diff
@@ -103,7 +96,7 @@ public void instanceIdShouldEqualHostnamePlusSuffix() throws UnknownHostExceptio
 				final MapConfiguration config = new MapConfiguration(map);
         final StandardJanusGraph graph = new StandardJanusGraph(new GraphDatabaseConfigurationBuilder().build(new CommonsConfiguration(config)));
         assertEquals(graph.openManagement().getOpenInstances().size(), 1);
-        assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName() + "1");
+        assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName() + "1(current)");
```

Comment:
Please change the line 
`assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName() + "1(current)");`
to
`assertEquals(graph.openManagement().getOpenInstances().toArray()[0], Inet4Address.getLocalHost().getHostName() + "1");`